src/components/TaskBoard/index.jsx

```diff
@@ -22,30 +38,52 @@ const TaskBoard = () => {
     }
   }, [showCreateCard]);
 
+  console.log(taskLength);
   return (
     <div className={style.container}>
       <h1>Add Task</h1>
-      <div>
+      <div className={style.topBar}>
         <button
+          className={style.createButton}
           disabled={showCreateCard}
           onClick={() => setShowCreateCard(!showCreateCard)}
         >
+          <img className={style.addIcon} src={ADD} alt={ADD_ALT} />
           Create
         </button>
+        <div className={style.filterBar}>
+          <button onClick={() => setFilter(ALL)} className={style.filterButton}>
+            All
+          </button>
+          <button
+            onClick={() => setFilter(INCOMPLETE)}
+            className={style.filterButton}
+          >
+            Incomplete
+          </button>
+          <button
+            onClick={() => setFilter(COMPLETE)}
+            className={style.filterButton}
+          >
+            Complete
+          </button>
```

Comment:
Could we use a map for not repeating same things again ?
```suggestion
          const filterStates = [ FILTER_STATE_ALL, ...]
          filterStates.map(......<button
            onClick={() => setFilter(COMPLETE)}
            className={style.filterButton}
          >
            Complete
          </button>)
          
```
Please correct the format I have provided.